ONSdigital/eq-questionnaire-runner
# fix `mobilenumber` input type width

### What is the context of this PR?
The `mobilenumber` type input field is only wide enough for 8 characters so will not fit a full mobile number especially if you suffix it with "+44". This will up the width to 15

### How to review 
- Open `test_multiple_answers` schema
- Enter "+44 0000 000000" into the mobile number field and see that the mobile number field is now wide enough for a full mobile number

### Checklist
